packages/cardhost/app/services/css-mode-toggle.js

```diff
@@ -0,0 +1,7 @@
+import Service from '@ember/service';
+
+export default class CssModeToggleService extends Service {
+  setEditingCss(value) {
```

Comment:
`editingCss` should probably be `@tracked`, though we have duplicated state between the query params and this service.